tstest/integration/vms/vms_test.go

```diff
@@ -294,14 +314,27 @@ func TestVMIntegrationEndToEnd(t *testing.T) {
 	}
 
 	distros := []Distro{
+		// NOTE(Xe): If you run into issues getting the autoconfig to work, comment out all the other distros and uncomment this one.
```

Comment:
Ni: here and elsewhere, can you wrap comments at 80. It makes code readability easier IMO.

IMO this is in line with https://github.com/golang/go/wiki/CodeReviewComments#line-length